src/app/modules/item/components/thread/thread.component.html

```diff
@@ -10,6 +10,7 @@
         <alg-thread-message
           class="thread-message"
           [event]="event"
+          [userCache]="(userCache | async) || []"
```

Comment:
I've found that it does subscribe as much as events count, as result I guess the problem is the same as before (subscribe in each message).

What if we move `(userCache | async)` out of *ngFor?

Reply:
You are completely right!

Reply:
fixed